From 3aff784811678c6848816787dc10920400beb376 Mon Sep 17 00:00:00 2001
From: Destrudos6 <3411622425@qq.com>
Date: Thu, 22 Oct 2026 12:57:51 +0800
Subject: [PATCH 2/4] Remove the leftover SoundCloudGoPlusContentException

The service removal patch deleted the SoundCloud service package but
missed this exception class in the shared exceptions directory; it
has no references left.
---
 .../exceptions/SoundCloudGoPlusContentException.java  | 11 -----------
 1 file changed, 11 deletions(-)
 delete mode 100644 extractor/src/main/java/org/schabi/newpipe/extractor/exceptions/SoundCloudGoPlusContentException.java

diff --git a/extractor/src/main/java/org/schabi/newpipe/extractor/exceptions/SoundCloudGoPlusContentException.java b/extractor/src/main/java/org/schabi/newpipe/extractor/exceptions/SoundCloudGoPlusContentException.java
deleted file mode 100644
index 072624b5..00000000
--- a/extractor/src/main/java/org/schabi/newpipe/extractor/exceptions/SoundCloudGoPlusContentException.java
+++ /dev/null
@@ -1,11 +0,0 @@
-package org.schabi.newpipe.extractor.exceptions;
-
-public class SoundCloudGoPlusContentException extends ContentNotAvailableException {
-    public SoundCloudGoPlusContentException() {
-        super("This track is a SoundCloud Go+ track");
-    }
-
-    public SoundCloudGoPlusContentException(final Throwable cause) {
-        super("This track is a SoundCloud Go+ track", cause);
-    }
-}
-- 
2.54.0.windows.1

